### PR TITLE
VZ-6982: update fluentd to latest version

### PR DESCRIPTION
**Pull request by adiforluls**

### CODE REVIEW

File: content/en/docs/setup/prereqs.md [hunk 1]
```diff
@@ -50,30 +50,31 @@ Verrazzano supports all versions of Helidon.  For more information, see [Helidon
...
+| Fluentd                      | 1.14.5       | Collects logs and sends them to OpenSearch.                                              |
+| Grafana                      | 7.5.11       | Tool to help you examine, analyze, and monitor metrics.                                  |
+| Istio                        | 1.13.2       | Service mesh that layers transparently onto existing distributed applications.           |
+| Jaeger                       | 1.34.1       | Distributed tracing system for monitoring and troubleshooting distributed systems.       |
+| Keycloak                     | 15.0.2       | Provides single sign-on with Identity and Access Management.                             |
+| Kiali                        | 1.42.0       | Management console for the Istio service mesh.                                           |
+| kube-state-metrics           | 2.4.2        | Provides metrics about the state of Kubernetes API objects.                              |
+| MySQL                        | 8.0.28       | Open source relational database management system used by Keycloak.                      |
```
Comment (bhatpmk) [inline content/en/docs/setup/prereqs.md:66]: The correct version of mysql is 8.0.29, coherence operator is 3.2.6. Can you please update all the components wherever a change is required ?

Comment (adiforluls) [reply content/en/docs/setup/prereqs.md]: Fixed

Comment (MelwynPaul) [reply content/en/docs/setup/prereqs.md]: Thank you. 
I will update the Release Notes to list these components under v1.4.0: _Fluentd_, _mysql_, and _coherence operator_
Info: The ReadMe files (README_FULL and README_LITE.md) already reflect the latest versions.

**Review by kminder — APPROVED** [review]
Approved.

**Review by MelwynPaul — APPROVED** [review]

**Pull request merged**